src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Lookup.cs

```diff
@@ -45,6 +45,11 @@ public override bool CheckTypes(TexlNode[] args, DType[] argTypes, IErrorContain
 
             var fValid = base.CheckTypes(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
 
+            if (!fValid)
+            {
+                return false;
+            }
```

Comment:
Why this?

This could be significant because it skips potential logic for determining a returnType. 
We still want returnType even if CheckTypes returns false (@bryancroteau-MSFT )

Reply:
This change wasn't supposed to be in this PR, thanks for catching it. Reverted